routes/store.php

```diff
@@ -67,21 +67,32 @@
     // Need to write this route this way to use a policy.
     // Registration wasn't being passed through for a Gate based trick.
 
+    // TODO middleware group this lot under can:view.
+
     Route::get(
         '/registrations/{registration}/voucher-manager',
         'BundleController@create'
     )
     ->name('store.registration.voucher-manager')
     ->middleware('can:view,registration');
 
-    // puts (and replaces!) the currentBundle of vouchers!
+    // PUTS (and replaces!) the currentBundle of vouchers!
```

Comment:
Is the `POST registrations/{id}/vouchers [start, end]` coming in a later PR?

Reply:
Yes, this is just the delete endpoint